06-developer-software-guide.Rmd

```diff
@@ -39,11 +39,82 @@ files, outputs, and build and debug environments. The most popular IDEs
  remotes::install_github("ManuelHentschel/vscDebugger")
  ```
 
+ To improve the plot viewer when creating plots in `R`, install the `httpgd` package:
+
+ ```{r eval=FALSE}
+ install.packages("httpgd")
+ ``` 
+
+A number of optional settings that could be added to the user settings (settings.json) file in vscode to improve the usability of R in VS code.
```

Comment:
Maybe provide instructions on where the user settings file is located? This link could be helpful: https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations. I found a few settings.json files in different places on my laptop and the correct settings.json is at %APPDATA%\Roaming\Code\User\settings.json